package.json

```diff
@@ -138,9 +138,6 @@
     "typescript": ">= 4.2.x <= 4.8.x"
   },
   "peerDependenciesMeta": {
-    "graphql": {
-      "optional": true
```

Comment:
I'd really prefer to not have msw in my dev bundle. There's potential that I could be using as a stopgap in prod until some micro-services are completed by another team.

Reply:
> I'd really prefer to not have msw in my dev bundle

How do you see it getting into your code then? Did you mean "prod" bundle?

MSW is and will always be a part of your dev bundle. Well, as long as you tell it to (use conditional require as suggested in the docs). My point here was that MSW won't ever get to a prod bundle unless that's your intention. And since it's not a user-facing KBs of code, it doesn't matter how many dependencies it installs, within the reason, of course.

Reply:
Sorry yes, my prod bundle. I had an issue that was closed (https://github.com/mswjs/msw/issues/1373) where graphql was in my prod bundle even though I'm not using it anywhere.

Reply:
I'd be fine with returning it as a dependency.  I think the peer strategy here should install a version at the top of the semver range as well, which _shouldn't_ require users import it directly in this case either, but a plain dependency is fine I think

> Those who don't use GraphQL won't have to make any changes to their projects. GraphQL will be installed as MSW's dependency but won't get into their bundle unless they A) use it; B) we code-split correctly.

Re this - I don't believe that's true currently - msw isn't shipping esm, and ships a single file bundle, so I doubt any tools could correctly remove/shake graphql from any consumer bundles (maybe possible, but offhand I don't think so).